vrc-get-gui/locales/zh_cn.json5

```diff
@@ -82,6 +82,7 @@
     "installed {{name}} version {{version}}": "已安装 {{name}} 版本 {{version}}",
     "removed {{name}}": "已删除 {{name}}",
     "upgraded all packages": "升级所有软件包",
+    "applied changes in bulk": "批量应用变化。",
```

Comment:
@lonelyicer could you check the localization?

Reply:
We can do that in another PR